packages/types/src/Metadata/v6/latest.substrate.v6.json

```diff
@@ -38,13 +38,36 @@
               "name": "AllExtrinsicsWeight",
               "modifier": "Optional",
               "type": {
-                "PlainType": "u32"
+                "PlainType": "Weight"
```

Comment:
This is going to create some issues down the line as soon as Babeb comes online. Babe also has weights, defined as `u64`, but the above is `u32`